Form-Controls/styles.css

```diff
@@ -0,0 +1,127 @@
+:root {
+    --white:#FFF;
+    --black:#000;
+    --db:#E6DACD;
+    --lb:#F4ECE6;
+    --rb:#0150FD;
+    --header-font: 'Poppins', sans-serif;
+    --gen-font: 'Quicksand', sans-serif;
+}
+
+body {
+    margin: 0;
+}
+
+header {
+    background: var(--white);
+    height: 100px;
```

Comment:
In general, avoid setting explicit heights on elements. It gets more and more difficult to manage the more complex your design. Instead of setting fixed sizes, typically you will allow the text to control the height of a box, and set minimums if content may not be present (for exampe: users are adding the content)